app/jobs/models.py

```diff
@@ -0,0 +1,91 @@
+from datetime import timedelta
+
+from django.conf import settings
+from django.db import models
+from django.urls import reverse
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
+
+# Create your models here.
+
+JOB_TYPE_CHOICES = (
+    ('full_time', _('Full-Time')),
+    ('part_time', _('Part-Time')),
+    ('contract', _('Contract')),
+    ('intern', _('Intern')),
+)
+
+
+def get_expiry_time():
+    return timezone.now() + timedelta(days=30)
+
+
+class Job(models.Model):
+    title = models.CharField(
+        verbose_name=_('Title'), max_length=200, null=False, blank=False
+    )
+    description = models.TextField(verbose_name=_('Description'))
+    github_profile_link = models.URLField(
```

Comment:
Github profiles should link directly back to `dashboard.Profile` versus char/url storage of text representing a `Profile`.